sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -540,16 +700,205 @@ def from_predictions(
+            created.
+
+        curve_kwargs : dict or list of dict, default=None
+            Dictionary with keywords passed to the matplotlib's `plot` function
+            to draw the individual precision-recall curves. If a list is provided, the
+            parameters are applied to the precision-recall curves of each CV fold
+            sequentially. If a single dictionary is provided, the same
+            parameters are applied to all precision-recall curves.
+
+        plot_chance_level : bool, default=False
+            Whether to plot the chance level lines.
+
+        chance_level_kwargs : dict, default=None
+            Keyword arguments to be passed to matplotlib's `plot` for rendering
+            the chance level lines.
+
+        despine : bool, default=False
+            Whether to remove the top and right spines from the plot.
+
+        Returns
+        -------
+        display : :class:`~sklearn.metrics.PrecisionRecallDisplay`
+
+        See Also
+        --------
+        PrecisionRecallDisplay.from_predictions : Plot precision-recall curve
+            using estimated probabilities or output of decision function.
+        PrecisionRecallDisplay.from_estimator : Plot precision-recall curve
+            using an estimator.
+        precision_recall_curve : Compute precision-recall pairs for different
+            probability thresholds.
+        average_precision_score : Compute average precision (AP) from prediction scores.
+
+        Notes
+        -----
+        The average precision (cf. :func:`~sklearn.metrics.average_precision_score`)
+        in scikit-learn is computed without any interpolation. To be consistent
+        with this metric, the precision-recall curve is plotted without any
+        interpolation as well (step-wise style).
+
+        You can change this style by passing the keyword argument
+        `drawstyle="default"`. However, the curve will not be strictly
+        consistent with the reported average precision.
+
+        Examples
+        --------
+        >>> import matplotlib.pyplot as plt
+        >>> from sklearn.datasets import make_classification
+        >>> from sklearn.metrics import PrecisionRecallDisplay
+        >>> from sklearn.model_selection import cross_validate
+        >>> from sklearn.svm import SVC
+        >>> X, y = make_classification(random_state=0)
+        >>> clf = SVC(random_state=0)
+        >>> cv_results = cross_validate(
+        ...     clf, X, y, cv=3, return_estimator=True, return_indices=True)
+        >>> PrecisionRecallDisplay.from_cv_results(cv_results, X, y)
+        <...>
+        >>> plt.show()
+        """
+        pos_label_ = cls._validate_from_cv_results_params(
+            cv_results, X, y, sample_weight=sample_weight, pos_label=pos_label
+        )
+
+        precision_folds, recall_folds, ap_folds, prevalence_pos_label_folds = (
+            [],
+            [],
+            [],
+            [],
+        )
```

Comment:
Lint did this and I don't think it looks great but I have no suggestions on how to fix :shrug: